packages/components/src/tools-panel/tools-panel-header/hook.js

```diff
@@ -23,11 +23,14 @@ export function useToolsPanelHeader( props ) {
 	}, [ className ] );
 
 	const { menuItems } = useToolsPanelContext();
-	const hasMenuItems = !! Object.entries( menuItems ).length;
+	const menuItemsArray = Object.entries( menuItems );
+	const hasMenuItems = !! menuItemsArray.length;
+	const hasSelectedMenuItems = menuItemsArray.some( ( [ , isSelected ] ) => isSelected );
```

Comment:
Looks like prettier is complaining that this is all on one line. Locally, this seems to do the trick:

```suggestion
	const hasSelectedMenuItems = menuItemsArray.some(
		( [ , isSelected ] ) => isSelected
	);
```